src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -4903,6 +4903,15 @@ public unsafe bool IsReplicationCertificateByPublicKeyPinningHash(ClusterOperati
             return false;
         }
 
+        public static bool TryReadNodeTag(ServerStore serverStore, out string nodeTag)
+        {
+            using (serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+                nodeTag = RachisConsensus.ReadNodeTag(context);
+
+            return nodeTag != RachisConsensus.InitialTag;
```

Comment:
Why do we compare the value we read with `RachisConsensus.InitialTag;`?

Reply:
To track the instance from which instruments come from, we pass a node tag as `instanceId`. However, if you have a fresh RavenDB instance and the cluster tag is not yet available, we skip the OpenTelemetry initialization.

Reply:
Maybe we should log that information? That we skipped that

Reply:
Added